### ISSUE TITLE
Edit breadcrumb text

### ISSUE THREAD

**Issue by vrybkin** [timelyportfolio/sunburstR#53]
Hi Kent and all,

Is it possible to modify the text of each breadcrumb element that is displayed at the very top of the plot? An example could be suppose that I am looking at a sequence of things, but I'd also like to know the duration of every element in the sequence. That is, if I am looking at the sequence A -> B -> C, I'd like to know the mean duration of A (when A is the first element in the sequence), the mean duration of B (when B is the second element in the sequence) and the mean duration of C (when C is the third element in the sequence). So basically, I'd like to know the mean duration (or any arbitrary statistic) for every element in every sequence. My thought was that if I could edit the breadcrumb text, I could somehow paste that information in there after computing it myself. But, I haven't an idea of how to access that information, as the "explanation" parameter is for the center of the sunburst plot only. 

What do you think, and do you have any suggestions?

Thanks for your time again!

**Comment by vrybkin** [comment]
I have resolved this issue using add_shiny, works well! My only thought would be that it would be nice to be able to click on one of the slices and have it stay highlighted, thereby keeping the input$NAME_mouseover "engaged" even if the mouse cursor is off of the figure. But - I understand if that is a pain.

**Comment by timelyportfolio** [comment]
I agree, but I have struggled with how to instruct users of the click functionality in a generalized setting.  Adding is not overly difficult, since we would just need to maintain a toggle state for clicked/unclicked.  But then we would need to decide if we want to support ctrl+click and shift+click.

**Comment by vrybkin** [comment]
Thanks for your reply, @timelyportfolio, that makes sense. For what it's worth, as a user I would not expect for control+click and shift+click to do anything, and just the clicking functionality would go a long ways for me. But, I definitely understand where you're coming from, in terms of generalization.
